File: abc/abc147/a.java
// https://atcoder.jp/contests/abc147/submissions/41617872

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    int a = stdIn.nextInt();
    int b = stdIn.nextInt();
    int c = stdIn.nextInt();

    System.out.println(a + b + c <= 21 ? "win" : "bust");
  }
}

File: abc/abc147/a.cs
// https://atcoder.jp/contests/abc147/submissions/46913445

class A
{
  static void Main()
  {
    string[] a = Console.ReadLine().Split(" ");
    
    int sum = 0;
    for(int i = 0; i < 3; i++)
    {
      sum += int.Parse(a[i]);
    }

    if(sum <= 21)
    {
      Console.Write("win");
    }
    else
    {
      Console.Write("bust");
    }
  }
}

